stickynotes/stickynotes_callbacks.c

```diff
@@ -22,170 +22,208 @@
+void
+popup_toggle_lock_cb (GtkToggleAction *action,
+                      StickyNote      *note)
 {
-	stickynote_set_locked(note, gtk_toggle_action_get_active(action));
+    stickynote_set_locked (note,
+                           gtk_toggle_action_get_active (action));
 }
 
 /* Popup Menu Callback : Change sticky note properties */
-void popup_properties_cb(GtkWidget *widget, StickyNote *note)
+void
+popup_properties_cb (GtkWidget  *widget,
+                     StickyNote *note)
 {
-	stickynote_change_properties(note);
+    stickynote_change_properties (note);
 }
 
 /* Properties Dialog Callback : Apply title */
-void properties_apply_title_cb(StickyNote *note)
+void
+properties_apply_title_cb (StickyNote *note)
 {
-	stickynote_set_title(note, gtk_entry_get_text(GTK_ENTRY(note->w_entry)));
+    stickynote_set_title (note,
+                          gtk_entry_get_text (GTK_ENTRY (note->w_entry)));
 }
 
 /* Properties Dialog Callback : Apply color */
-void properties_apply_color_cb(StickyNote *note)
+void
+properties_apply_color_cb (StickyNote *note)
 {
-	char *color_str = NULL;
-	char *font_color_str = NULL;
-	
-	if (!gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(note->w_def_color)))
-	{
-		GdkRGBA color, font_color;
+    char *color_str = NULL;
+    char *font_color_str = NULL;
+
+    if (!gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON (note->w_def_color))) {
+        GdkRGBA color, font_color;
+
+        gtk_color_chooser_get_rgba (GTK_COLOR_CHOOSER (note->w_color),
+                                                       &color);
+        gtk_color_chooser_get_rgba (GTK_COLOR_CHOOSER (note->w_font_color),
+                                                       &font_color);
 
-		gtk_color_chooser_get_rgba (GTK_COLOR_CHOOSER (note->w_color), &color);
-		gtk_color_chooser_get_rgba (GTK_COLOR_CHOOSER (note->w_font_color), &font_color);
+        color_str = gdk_rgba_to_string (&color);
+        font_color_str = gdk_rgba_to_string (&font_color);
+    }
 
-		color_str = gdk_rgba_to_string (&color);
-		font_color_str = gdk_rgba_to_string (&font_color);
-	}
-	
-	stickynote_set_color (note, color_str, font_color_str, TRUE);
+    stickynote_set_color (note, color_str,
+                          font_color_str, TRUE);
 
-	g_free (color_str);
-	g_free (font_color_str);
+    g_free (color_str);
+    g_free (font_color_str);
 }
 
 /* Properties Dialog Callback : Apply font */
-void properties_apply_font_cb(StickyNote *note)
+void properties_apply_font_cb (StickyNote *note)
 {
-	const gchar *font_str = NULL;
-	
-	if (!gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(note->w_def_font)))
-	{
-		font_str = gtk_font_button_get_font_name (
-				GTK_FONT_BUTTON (note->w_font));
-	}
+    const gchar *font_str = NULL;
 
-	stickynote_set_font(note, font_str, TRUE);
+    if (!gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON (note->w_def_font))) {
+        font_str = gtk_font_button_get_font_name (
+            GTK_FONT_BUTTON (note->w_font));
```

Comment:
same here

Reply:
I guess i can remove the parentheses from the one liner `if` condition.

Reply:
Done.